radis/lbl/factory.py

```diff
@@ -505,38 +508,39 @@ def __init__(
         # Init variables
         # --------------
         # Get molecule name
-        if not is_atom(species): 
-            if isinstance(species, int):
-                species == get_molecule(species)
-            if species is not None:
-                if (
-                    species
-                    not in MOLECULES_LIST_EQUILIBRIUM + MOLECULES_LIST_NONEQUILIBRIUM
-                ):
-                    raise ValueError(
-                        "Unsupported molecule: {0}.\n".format(species)
-                        + "Supported molecules are:\n - under equilibrium: {0}".format(
-                            MOLECULES_LIST_EQUILIBRIUM
-                        )
-                        + "\n- under nonequilibrium: {0}".format(
-                            MOLECULES_LIST_NONEQUILIBRIUM
+        if molecule is not None and species is not None:
```

Comment:
Fix molecule and/or species when reading the function parameters.

Then use one only through the whole code. Here you're still using both it makes the code hard to read. 

You can rename the internal name to species to be very clear.